packages/theme-default/src/components/Overview/index.tsx

```diff
@@ -273,16 +291,22 @@ export function Overview(props: {
       .filter(Boolean);
   }, [groups, query]);
 
+  const overviewTitle = title || 'Overview';
+
   return (
     <div className="overview-index mx-auto px-8">
       <div className="flex flex-col sm:flex-row items-start sm:items-center justify-between">
-        {!title && (
-          <h1 className="text-3xl leading-10 tracking-tight mb-4 sm:mb-0">
-            Overview
-          </h1>
-        )}
+        <h1 className="text-3xl leading-10 tracking-tight mb-4 sm:mb-0">
+          {overviewTitle}
+        </h1>
         {/* Added search input */}
-        <SearchInput query={query} setQuery={setQuery} searchRef={searchRef} />
+        <SearchInput
+          query={query}
+          setQuery={setQuery}
+          searchRef={searchRef}
+          apiFilterText={apiFilterText}
+          apiFilterPlaceholderText={apiFilterPlaceholderText}
+        />
       </div>
       {content}
```

Comment:
If user do not use title in frontmatter

```mdx
---
overview: true
---

# overview

content
```

the `<h1 />` and `<SearchInput />` will display like below

![image](https://github.com/user-attachments/assets/308ca1aa-e33b-44ac-b20c-d0b8d66a4aee)

Can we compatible with this situation?

Reply:
Yeah, I'll try it.